build.gradle

```diff
@@ -60,8 +62,12 @@ buildscript {
         classpath "gradle.plugin.io.morethan.jmhreport:gradle-jmh-report:0.9.0"
         classpath "org.jlleitschuh.gradle:ktlint-gradle:$ktlint_gradle_version"
         classpath "org.jetbrains.kotlinx:atomicfu-gradle-plugin:$atomicfuVersion"
+        classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.4.6'
     }
 }
+dependencies {
```

Comment:
i'm not sure that this is being applied to all submodules, i think you need to move it below inside `allProjects` together with `apply plugin: 'ru.vyarus.animalsniffer'`

Reply:
Ok, moved